src/platforms/salesforce/schemas.ts

```diff
@@ -80,28 +97,31 @@ export const salesforceContactUpdate = validators.object({
 // -
 // Account
 // -
-export const salesforceAccount = validators.object({
-  Id: z.string(),
-  Name: z.string(),
-  Description: z.string(),
-  Industry: z.string(),
-  AnnualRevenue: z.string(),
-  NumberOfEmployees: z.string(),
-  Website: z.string(),
-  BillingAddress: z.object({
-    street: z.string(),
-    city: z.string(),
-    state: z.string(),
-    postalCode: z.string(),
-    country: z.string(),
-  }),
-  Phone: z.string(),
-  CreatedDate: validators.date(),
-  LastModifiedDate: validators.date(),
-  OwnerId: z.string(),
-  Contacts: z.object({ records: z.array(z.object({ Id: z.string() })) }),
-  Opportunities: z.object({ records: z.array(z.object({ Id: z.string() })) }),
-});
+export const salesforceAccount = validators
+  .object({
+    Id: z.string(),
+    Name: z.string(),
+    Description: z.string(),
+    Industry: z.string(),
+    AnnualRevenue: z.string(),
+    NumberOfEmployees: z.string(),
+    Website: z.string(),
+    BillingAddress: z.object({
+      street: z.string(),
```

Comment:
Just confirming these are all supposed to be lower case?

Reply:
Yeah, it's all lower-case (copied over from monorepo and double checked online)